src/test/groovy/com/stormpath/tck/util/CookieUtils.groovy

```diff
@@ -22,6 +22,6 @@ class CookieUtils {
 
     public static boolean isCookieDeleted(Cookie cookie) {
         def now = new Date()
-        return cookie.expiryDate < now
+        return cookie == null || cookie.expiryDate < now
```

Comment:
If cookie is null, it is not deleted. Cookies can only be deleted by setting the expiry date to before the current time. 

https://msdn.microsoft.com/en-us/library/ms178195.aspx